src/engine/localevent.h

```diff
@@ -306,6 +306,16 @@ class LocalEvent
         }
     }
 
+    bool isDragInProgress() const
+    {
+        return modes & DRAG_ONGOING;
+    }
+
+    void RegisterDrag()
```

Comment:
We can name the new method according to the [suggested code style](https://github.com/ihhub/fheroes2/wiki/Code-style) :)
```suggestion
    void registerDrag()
```